ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -295,13 +296,18 @@ const VulnMgmtCves = ({
         hasReadWriteAccess('VulnerabilityManagementApprovals') &&
         hasReadWriteAccess('VulnerabilityManagementRequests');
 
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isUnifiedDeferralEnabled = isFeatureFlagEnabled('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL');
+
     const [selectedCveIds, setSelectedCveIds] = useState([]);
     const [bulkActionCveIds, setBulkActionCveIds] = useState([]);
 
     const workflowState = useContext(workflowStateContext);
 
     const cveType = workflowState.getCurrentEntityType();
 
+    const shouldRenderGlobalSnooze = !isUnifiedDeferralEnabled || cveType !== entityTypes.IMAGE_CVE;
```

Comment:
I just wanted to make sure I'm getting the logic right. I had to look at it a bit. So if `ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL` is disabled, then the first part of this conditional will be `true`. That means the second part wouldn't matter, right? So can we potentially have `ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL` as disabled and the CVE type is not an Image CVE, and it'll still render? Wouldn't you only want to show for Image CVEs or am I mistaken?

Reply:
The components that this conditional renders are shown for image, node, and platform CVEs. The logic I was going for was:
- If the UNIFIED_DEFERRALS flag is not set, always render the snooze UI
- If the flag _is_ set, only render the snooze UI when the entity is something other than "IMAGE_CVE"

I actually tried this out as `!(isUnifiedDeferralEnabled && cveType === entityTypes.IMAGE_CVE)` for readability, but switched back after looking at it. Do you think that reads better than what I have here?